### ISSUE TITLE
Support Symfony 7

### ISSUE THREAD

**Issue by tacman** [tightenco/collect#346]
It's only for vardumper, but in general I think it's a good idea to stop supporting EOL versions of Symfony.  I'll submit a PR so it can at least be installed.

**Comment by GrahamCampbell** [comment]
This whole package is EOL, anyway. One should use https://packagist.org/packages/illuminate/collections instead.

**Comment by tacman** [comment]
Yeah, I have a bundle that uses a library that use this library, it hasn't been updated since 2020, I was trying to find a way get it installed without updating everything, but since 3 of the 4 outstanding issues are mine, I guess I'm going to fork it and make the changes.

https://github.com/freearhey/wikidata/issues

On the plus side, I do want to learn the collections package, especially now that it's less tied to Lavarel.  Thanks for releasing and maintaining it!
